app/views/home/partials/_nav.html.erb

```diff
@@ -0,0 +1,28 @@
+<section>
+  <div>
+    <div>
+      <%# SVG %>
+    </div>
+    
+    <ul>
+      <li>
+        <a href="#products" data-controller="scroll-to">PRODUCTS</a>
+      </li>
+      <li>
+        <a href="#pricing" data-controller="scroll-to">PRICING</a>
+      </li>
+      <li>
+        <a href="about-us" data-controller="scroll-to">ABOUT US</a>
+      </li>
```

Comment:
In this section you can use as well the rails helper for `link_to` just like you did below. To keep it consistent i would suggest using them on both cases, what do you think?

Reply:
Thanks, that solution was better than the one we are implementing.